pkg/sensors/tracing/kprobe_copyfd_test.go

```diff
@@ -52,7 +53,11 @@ func TestCopyFd(t *testing.T) {
 		data := map[string]string{
 			"MatchedPID": pid,
 		}
-		specName, err := testutils.GetSpecFromTemplate("copyfd.yaml.tmpl", data)
+		templatePath := "copyfd511.yaml.tmpl"
+		if kernels.IsKernelVersionLessThan("5.11.0") {
+			templatePath = "copyfd510.yaml.tmpl"
```

Comment:
nit: would it make sense to rename the file to `copyfd-__fd_install` or something?

Can we also add a comment here that describes why we need a second file? This part from the commit message, for example:

> For kernels <= 5.10, dup syscall calls fd_install, which calls
> __fd_install. fd_install is inlined and if we hook there we miss
> the dup event.